RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -157,21 +159,4 @@ private extension CustomerCenterView {
 
 }
 
-#if DEBUG
-
-@available(iOS 15.0, *)
-@available(macOS, unavailable)
-@available(tvOS, unavailable)
-@available(watchOS, unavailable)
-struct CustomerCenterView_Previews: PreviewProvider {
-
-   static var previews: some View {
```

Comment:
It'd be nice to be able to keep this if it's possible - maybe we could mock out a `PurchaseInformation` object for the view model to use?

Reply:
good idea, I think I can mock the three different cases (Active, no active, and wrong platform)

Reply:
I ended up adding just one preview for a monthly apple product, the other ones were becoming a bit of hell to support